src/web/handlers/FlingGestureHandler.ts

```diff
@@ -73,11 +71,18 @@ export default class FlingGestureHandler extends GestureHandler {
     };
 
     // list of alignments to all activated directions
-    const alignmentList = Object.values(Directions).map((direction) =>
-      getAlignment(direction, this.minimalAlignmentCosine)
+    const axialAlignmentList = axialDirectionsList.map((direction) =>
+      getAlignment(direction, AXIAL_DEVIATION_COSINE)
+    );
+
+    const diagonalAlignmentList = axialDirectionsList.map((direction) =>
+      getAlignment(direction, DIAGONAL_DEVIATION_COSINE)
     );
 
-    const isAligned = alignmentList.some(Boolean);
+    const isAligned = axialAlignmentList
+      .concat(diagonalAlignmentList)
+      .some(Boolean);
```

Comment:
```suggestion
    const isAligned = axialAlignmentList.some(Boolean) || diagonalAlignmentList.some(Boolean);
```

`concat` creates a new array each time with values from both sources copied